runbooks/serve_opg_runbook.md

```diff
@@ -0,0 +1,47 @@
+---
+category: runbooks
+expires: 2019-12-01
+---
+# Serve OPG Runbook
+
+## Description
+
+Serve OPG allows the Court of Protection to serve OPG deputyship orders directly into our case management system Sirius
+
+## Incident Response Hours
+
+Currently we respond to incidents within the hours outlined below:
+
+    9am-5pm, Monday to Friday, excluing bank holidays
+
+## Incident contact details
+
+If you need to contact us to raise an incident or discuss issues that will imminently have an impact on our service you should contact them via PagerDuty. This will ensure members of the team are immediately notifed.
+
+    serve-opg@moj-digital-tools.pagerduty.com
+
+## URLs
+
+### Service URLs
+    
+    https://development.serve.opg.service.justice.gov.uk
+    https://preproduction.serve.opg.service.justice.gov.uk
+    https://serve.opg.service.justice.gov.uk
+
+### Other URLs
+
+#### Github
+    
+    https://github.com/ministryofjustice/serve-opg  
+    https://github.com/ministryofjustice/serve-opg-infrastructure  
+
+## Hosting environment
+
+Currently ServeOPG is hosted in AWS using ECS, an architecture diagram can be found [here](https://github.com/ministryofjustice/serve-opg/tree/master/docs/architecture)
+
+
+## Service Team (non-urgent issues)
+
+For non-urgent issues you can raise an Issue on our GitHub repository and a member of the team will respond.
+
+    https://github.com/ministryofjustice/serve-opg/issues
```

Comment:
email address for non urgent issues:  opgallocations@publicguardian.gov.uk

Reply:
Hey Sean, This needs to be a contact for us, not the team who use the service.